### PR TITLE
add deaccumulate option

### LINKED ISSUE #319: Add deaccumulate option
### RVT
The `:Deaccumulate` command, if included will take cumulative precipitation values stored in the NetCDF file (such as are output by some weather models) and convert these to precipitation rates, in mm/d.

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes #319 

Changes:

* Added `:Deaccumulate` option to RVT template.

### CODE REVIEW

File: raven/models/rv.py [hunk 1]
```diff
@@ -309,6 +311,15 @@ def linear_transform(self, value):
 
         self._linear_transform = value
 
+    @property
+    def deaccumulate(self):
+        """Convert cumulative precipitation to precipitation rate (mm/d)."""
+        return ":Deaccumulate" if self._deaccumulate is True else ""
```
Comment (cjauvin) [inline raven/models/rv.py:317]: Vraiment un nitpick pour nitpicker ici mais le `is True` n'est pas redondant?

**Review by cjauvin — APPROVED** [review]

File: raven/models/rv.py (hunk 1 above)
Comment (huard) [reply raven/models/rv.py:317]: Je voulais m'assurer que c'est bien un boolean, et pas juste 42.

**Review by cjauvin — APPROVED** [review]

File: raven/models/base.py [hunk 2]
```diff
@@ -542,8 +542,8 @@ def _assign_files(self, fns):
                                                    var_name=alt_name,
                                                    dimensions=ds[alt_name].dims,
                                                    units=ds[alt_name].attrs.get("units"),
+                                                   deaccumulate=ds[alt_name].attrs.get("GRIB_stepType", "") == "accum",
```
Comment (richardarsenault) [inline raven/models/base.py:545]: This line makes it that I cannot force the "deaccumulate" flag in model.rvt.pr.deaccumulate=True, if the flag is not in the file it returns "" and overwrites my <True> tag. Up until here, the flag is set to true, but then gets overwritten here. So I guess the best alternative is to add that flag when I generate the new netcdf for Raven from the forecasts?

Comment (richardarsenault) [reply raven/models/base.py:545]: Actually, I did just that, adding the GRIB_stepType attribute to 'accum' got it detected and works now.

Comment (huard) [reply raven/models/base.py:545]: Hum, but ideally we would not overwrite it. I'll patch something.

**Comment by richardarsenault** [comment]
This now works. I will merge now so I can use it in my hindcasting branch.

**Pull request merged**